packages/knip/src/plugins/cypress/helpers.ts

```diff
@@ -0,0 +1,31 @@
+import type { PluginOptions } from '#p/types/plugins.ts';
+import { isInternal, toAbsolute } from '#p/util/path.ts';
+import { load, resolveEntry } from '#p/util/plugin.ts';
+import type { CypressConfig } from './types.js';
+
+export const resolveDependencies = async (config: CypressConfig, options: PluginOptions) => {
+  const { reporter } = config;
+  const { configFileDir } = options;
+
+  const resolve = (specifier: string) => resolveEntry(options, specifier);
+
+  // Initialize the array of reporters with the initial reporter if present.
+  const reporters = reporter ? [reporter] : [];
+  // https://github.com/YOU54F/cypress-plugins/tree/master/cypress-multi-reporters#configuring-reporters
+  if (reporter === 'cypress-multi-reporters' && config.reporterOptions?.configFile) {
+    // Try to resolve the config file if present and attach the reporters listed in it.
+    const { configFile } = config.reporterOptions;
+    const configFilePath = toAbsolute(configFile, configFileDir);
+    if (isInternal(configFilePath)) {
+      const reporterConfig = await load(configFilePath);
+      if (typeof reporterConfig === 'object' && reporterConfig.reporterEnabled) {
+        const { reporterEnabled: reporterConcatenatedNames } = reporterConfig;
+        const reporterNames = reporterConcatenatedNames.split(', ');
```

Comment:
Maybe allow without spaces too? eg:

```
.split(/, ?/)
```

Reply:
Good call, looks like they use `.trim()` in their code (https://github.com/YOU54F/cypress-plugins/blob/master/cypress-multi-reporters/lib/MultiReporters.js#L50-L58). Let me try that here as well.

Reply:
Updated in [e038b50](https://github.com/webpro-nl/knip/pull/726/commits/e038b50a37eaaf610f03489b9bde1bdf379e03d1)